src/functions/severa/get-flextime-by-user/handler.ts

```diff
@@ -20,18 +22,40 @@ export const getFlextimeHandler: APIGatewayProxyHandler = async (event) => {
       };
     }
 
-    const flexTimeBySeveraId = await api.getFlextimeBySeveraUserId(severaUserId);
+    try {
+      await api.getKeywordIdForUser(severaUserId, "isSeveraOptIn");
+    } catch {
+      return {
+        statusCode: 403,
+        body: JSON.stringify({ message: "User has not opted in." }),
+      };
+    }
+
+    const severaUser = await api.getUser(severaUserId);
+
+    const testUserEmail = Config.get().testUser.email;
+    const keycloakEmail = testUserEmail || event.requestContext?.authorizer?.claims?.email;
```

Comment:
What about in production where we don't want to use the testUser email.
I'm concerned it would crash if the env is missing- so we need to make sure it doesn't (test locally what happens).